FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/physical/Robot.java

```diff
@@ -32,12 +33,12 @@
  * directory.
  */
 public class Robot extends IterativeRobot implements IRobot {
-	static final int LEFT_INDEX = 0;
-	static final int LEFT_SLAVE_INDEX = 1;
-	static final int RIGHT_INDEX = 2;
-	static final int RIGHT_SLAVE_INDEX = 3;
-	static final int INTAKE_INDEX = 4;
-	static final int ARM_INDEX = 5;
+	static final int LEFT_INDEX = 3;

```

Comment:
The actual robots use different indices on the motor controllers. This is what happens to work.